vet-service/src/test/java/com/petclinic/vet/presentationlayer/VetControllerUnitTest.java

```diff
@@ -566,6 +567,41 @@ void deleteEducationForVetByEducationId_ShouldSucceed() {
                 .deleteEducationByEducationId(vetId,educationId);
     }
 
+    @Test
+    void addEducationWithVetId_ValidValues_ShouldSucceed(){
+        EducationRequestDTO educationRequestDTO = EducationRequestDTO.builder()
+                .vetId(VET_ID)
+                .degree("Doctor of Veterinary Medicine")
+                .fieldOfStudy("Veterinary Medicine")
+                .schoolName("University of Montreal")
+                .startDate("2010")
+                .endDate("2014")
+                .build();
+
+        EducationResponseDTO educationResponseDTO = EducationResponseDTO.builder()
+                .educationId("3")
+                .vetId(VET_ID)
+                .degree("Doctor of Veterinary Medicine")
+                .fieldOfStudy("Veterinary Medicine")
+                .schoolName("University of Montreal")
+                .startDate("2010")
+                .endDate("2014")
+                .build();
+
+        when(educationService.addEducationToVet(VET_ID, Mono.just(educationRequestDTO))).thenReturn(Mono.just(educationResponseDTO));
+
+        client.post()
+                .uri("/vets/{vetId}/educations", VET_ID)
+                .bodyValue(educationRequestDTO)
+                .accept(MediaType.APPLICATION_JSON)
+                .exchange()
+                .expectStatus().isOk()
+                .expectHeader().contentType(MediaType.APPLICATION_JSON)
+                .expectBody();
```

Comment:
Could check using .jsonPath if the attributes of the body match the expected attributes of educationResponseDTO, but otherwise, looks good.